.github/labeler.yml

```diff
@@ -13,3 +13,7 @@ Dependencies:
 CI/CD:
 - changed-files:
   - any-glob-to-any-file: ['docker/**/*', '.github/**/*', '.ci/**/*']
+
+Testing:
```

Comment:
Both option are fine by me. If you don't find it useful, I can remove it.

Reply:
The problem I see with that label, is that every PR modifying code should add unit tests for that change, hence most of the PR will also modify the testing directory.
I don't think it will be then very descriptive having such label then.

However, I leave it up to you. You maybe saw that label as useful in other repositories. Do as you wish really. :)